### DISCUSSION TITLE
Hide AutoModerator (or pinned) comments

### DISCUSSION THREAD

**Discussion by ashut0shk** [Docile-Alligator/Infinity-For-Reddit#914] [Ideas]
Hi

Please add option to hide/collapse the AutoModerator comments, which are usually pinned to posts. 

A decent way could be to have the possibility to hide the pinned comments altogether, configurable from the Filter section of the app. 

Thanks

**Comment by juliloving** [comment]
Any updates on this? This feature is something I'm really interested in
